taxonomies/ingredients.txt

```diff
@@ -32355,10 +32361,11 @@ nl:ammonium- en natriumcarbonaten, ammonium carbonaten en natrium
 #
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
-
+en:fruit and vegetable
```

Comment:
I would prefer not to add entries for "something and something else".

Reply:
I added it to capture `fruit and vegetable juice`. I see it as a combined ingredient, not as two separate ones. Thus I made it a parent of `en:fruit` and `en:vegetable`. For other and-constructions I do the inverse, i.e. `en:fruit` and `en:vegetable` would be parents of `en:fruit and vegetable`.

Reply:
OK, in that case we should deal with "fruit and vegetable juice" in a different way, without making "fruit and vegetable" an ingredient. We will need to handle it when we do the parsing. Because otherwise we would need add countless combinations so that we can handle "mango and apple juice" etc.

We also can't have "fruit and vegetable" as a parent of "fruit" and "vegetable", otherwise all products that have either a fruit, or a vegetable, will have "fruit and vegetable" as an ingredient, which would be very confusing.